test/site/_transforms/service-worker-partials/index.js

```diff
@@ -12,7 +12,7 @@ describe('service-worker-partials', function() {
   let $;
 
   beforeEach(function() {
-    const fixture = `<html lang="en"><head><title>Test title</title></head><body><div id="content"><div class="guide-landing-page"><div>Hello</div></div><div>Bonus sibling</div></div></body></html>`;
+    const fixture = `<html lang="en"><head><title>Test title</title><link rel="alternate" href="/feed.xml" type="application/atom+xml" data-title="web.dev feed"></head><body><div id="content"><div class="guide-landing-page"><div>Hello</div></div><div>Bonus sibling</div></div></body></html>`;
```

Comment:
would you mind adding a test that verifies that it returns a unique rss for the author and/or tags page? I think testing either is fine. Just something to assert that if the <link> has a different value we will not return /feed.xml